### PR TITLE
docs: storybook default values

### PR DESCRIPTION
**Pull request by anatoliiboiarshchykov-okta**
[OKTA-633500](https://oktainc.atlassian.net/browse/OKTA-633500)

## Summary

- Update default value for components in storybook
- Add **Required** in description

## Testing & Screenshots
Screenshot 2023-10-18 at 12 13 02
Screenshot 2023-10-18 at 20 57 47

- [x] I have confirmed this change with my designer and the Odyssey Design Team.

### CODE REVIEW

File: packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.stories.tsx [hunk 1]
```diff
@@ -71,7 +70,7 @@ const storybookMeta: Meta<typeof Banner> = {
     severity: {
       options: bannerSeverityValues,
       control: { type: "radio" },
-      description: "Determine the color and icon of the alert",
+      description: "<b>Required.</b> Determine the color and icon of the alert",
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.stories.tsx:73]: I thought there was a way to get Storybook to put a red asterisk. We might need to check their source code.

I know you put this in, but I don't wanna add `<b>Required.</b>` each time. It doesn't scale. Can we remove those?

File: packages/odyssey-storybook/src/components/odyssey-mui/Checkbox/Checkbox.stories.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ import { expect } from "@storybook/jest";
 import { axeRun } from "../../../axe-util";
 import type { PlaywrightProps } from "../storybookTypes";
 
-const storybookMeta: Meta<CheckboxProps> = {
+const storybookMeta: Meta<typeof Checkbox> = {
```
Comment (KevinGhadyani-Okta) [inline packages/odyssey-storybook/src/components/odyssey-mui/Checkbox/Checkbox.stories.tsx:25]: I thought using `CheckboxProps` was better for these. Any reason we're using `typeof Checkbox` now?

I know Storybook v7 changed some TypeScript pieces, so I'm thinking you can elaborate.

**Review by KevinGhadyani-Okta — COMMENTED** [review]
Overall, the changes look good. Just a few small comments.

File: packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.stories.tsx (hunk 1 above)
Comment (KevinGhadyani-Okta) [reply packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.stories.tsx:73]: I think I found it:
image

Source:
https://storybook.js.org/docs/vue/api/arg-types

File: packages/odyssey-storybook/src/components/odyssey-mui/Checkbox/Checkbox.stories.tsx (hunk 2 above)
Comment (anatoliiboiarshchykov-okta) [reply packages/odyssey-storybook/src/components/odyssey-mui/Checkbox/Checkbox.stories.tsx]: I forgot to roll it back, jus t was playing around the code a little bit

Comment (anatoliiboiarshchykov-okta) [reply packages/odyssey-storybook/src/components/odyssey-mui/Checkbox/Checkbox.stories.tsx]: Updated

File: packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.stories.tsx (hunk 1 above)
Comment (anatoliiboiarshchykov-okta) [reply packages/odyssey-storybook/src/components/odyssey-mui/Banner/Banner.stories.tsx]: Updated

**Review by KevinGhadyani-Okta — APPROVED** [review]
Woo thanks!

**Comment by KevinGhadyani-Okta** [comment]
I had to apply @anatoliiboiarshchykov-okta's patch myself because he lost Okta GitHub access. He was the one that made those changes 👍.

**Pull request merged**